scripts/tests/requirements.txt

```diff
@@ -5,6 +5,7 @@ colorama
 coloredlogs
 diskcache
 mypy==1.10.1
-pyyaml
 tabulate
-websockets
+
+-e ${PW_PROJECT_ROOT}/scripts/py_matter_idl
```

Comment:
This needs adjustments in `integrations/docker/images/chip-cert-bins/Dockerfile`.

Do not resolve this comment until problem is fixed!